### ISSUE TITLE
OTLP exporters should add their version information as a resource attribute

### ISSUE THREAD

**Issue by codeboten** [open-telemetry/opentelemetry-specification#2501]
As a result of the discussion in April 19th SIG meeting, it was suggested that it would be useful for debugging purposes that the OTLP exporter add version information to the resource attributes it emits.

**Comment by bogdandrutu** [comment]
No, it is about the version of the exporter component. We decided intentionally to not add the version of the `opentelemetry-proto` to disallow backends to do business decisions based on the proto version. This information can be used only for metrics/debugging purpose.

**Comment by alanwest** [comment]
Maybe worthy of a separate issue, but I think it also makes sense to add this information in the user-agent (or equivalent). Much in the same way that was done in the collector project https://github.com/open-telemetry/opentelemetry-collector/pull/3970.
